app/build.gradle.kts

```diff
@@ -201,18 +201,23 @@ dependencies {
 }
 
 tasks.register<Copy>("moveFromi18n") {
-    val locales = listOf("de", "es", "sv", "en-US")
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+    destinationDir = file("src/main/res")
 
+    val locales = listOf("de", "es", "sv", "en-US")
```

Comment:
Do we need to expand this with the other localizations that are included in Scribe-i18n?

Reply:
val  locales = file("src/main/assets/i18n/Scribe-i18n/values").listFiles()
        .findAll { it.isDirectory() }
        .collect { it.name }
        
could we use something like this